language-server/modules/langserver-core/src/test/java/org/ballerinalang/langserver/completion/TypeDescContextTest.java

```diff
@@ -51,6 +52,9 @@ public String getTestResourceDir() {
 
     @Override
     public List<String> skipList() {
-        return Arrays.asList();
+        ArrayList<String> skip = new ArrayList<>();
+        // Skip this test due to issue: https://github.com/ballerina-platform/ballerina-lang/issues/37039
+        skip.add("function_typedesc17.json");
```

Comment:
Skip this test due to issue: https://github.com/ballerina-platform/ballerina-lang/issues/37039

Reply:
Shall we use 
```
return Arrays.asList(
        "function_typedesc17.json"
    );
```
as in other LS tests.